libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -26,6 +28,17 @@ export class BookSearchComponent {
     private readonly fb: FormBuilder
   ) {}
 
+  ngOnInit(): void {
+    this.subscription.add(
```

Comment:
Here we only have one subscription so instead of using `this.subscription.add`, you can simply use 

`takeUntil(this.ngUnSubscribe$)`

and 

```
ngOnDestroy(): void {
    this.ngUnSubscribe$.next();
    this.ngUnSubscribe$.unsubscribe();
  }
```

Reply:
Also you can simplify it using

```
this.searchForm.get('term').valueChanges
```
this way you can use only `distinctUntilChanged ()` instead of whatever you have written currently.